### ISSUE TITLE
Very strange empty space at the bottom of the page in 20.0.2 

### ISSUE THREAD

**Issue by elpraga** [nextcloud/files_pdfviewer#269]
https://github.com/nextcloud/files_pdfviewer/issues/268 Got closed because someone complained I did no use a template.

There is a very unprofessionally looking glitch in the PDF viewer. I'm quite overwhelmed by work right now. So this is the last report I'm able to write in a while. If it doesn't meet your standards, just feel free to let your product looked half broken. 

### Steps to reproduce
1. click on a PDF

### Expected behaviour
There is no strange uncovered space at the bottom

### Actual behaviour
There is strange uncovered space at the bottom

### Server configuration

**Operating system**:
Ubuntu 20.04
**Web server:**
Apache
**Database:**
MariaDB
**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)
20.02
**Where did you install Nextcloud from:**
Have been upgrading since 15.0

**List of activated apps:**
Breeze Dark

**Comment by skjnldsv** [comment]
Hi,

Thanks for filling the template a little more.
The full list of apps is still missing. We're not asking just for fun, it helps us narrow your issue quicker. While you're `overwhelmed by work right now`, keep in mind having to look a tons of issues with missing informations makes us quite overwhelmed too. Hope you'll understand.

I cannot reproduce the issue, so I assume it's either a custom css (do you have any?), a custom theme (do you have any?) or the `Breeze Dark` theme (have you tried disabling it?)

Best regards

**Comment by elpraga** [comment]
```
keep in mind having to look a tons of issues with missing informations makes us quite overwhelmed too. Hope you'll understand.
```

Yes, I do.

```
I cannot reproduce the issue, so I assume it's either a custom css (do you have any?), a custom theme (do you have any?) or the Breeze Dark theme (have you tried disabling it?)
```

From your initial response, I had no idea that you cannot reproduce the issue. Knowing that helps me better understand your need for additional information. 

I do not use any custom theme (to my knowledge - I've been upgrading since Nextcloud 14 or so). **As far as I remember** I have not made any chanes to the theme CSS apart from changing the items in **Settings - Theme**. 

I even installed the **Custom CSS** app now to investigate if I happen to have any stale CSS lying around from any previous installs. (Note that the issue appeared long before installing the Custom CSS app). But the app did not reveal any CSS code used.

Is it possible that I changed the CSS manually way back in the NC14 days, and it only now started giving problems? If so, is there a way to check that?

Yes, I did try disabling Breeze Dark theme, but to no avail.

Note that I have not experienced this issue in NC19 and previous versions.

Thank you for the tip to use the Issue Template app - it helped.

### Server configuration


**Operating system**:
Linux 5.4.0-1025-raspi #28-Ubuntu SMP PREEMPT Wed Dec 9 17:10:53 UTC 2020 aarch64
**Web server:**
Apache/2.4.41 (Ubuntu) (apache2handler)
**Database:**
mysql 10.3.25
**PHP version:**
7.4.3
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, sodium, apache2handler, pdlib, mysqlnd, PDO, xml, apcu, bcmath, bz2, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, intl, json, exif, msgpack, mysqli, pdo_mysql, apc, posix, readline, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tokenizer, xmlreader, xmlwriter, xsl, zip, Phar, memcached, Zend OPcache
**Nextcloud version:** (see Nextcloud admin page)
20.0.4 - 20.0.4.0
**Where did you install Nextcloud from:**
Kept updating from I believe NC14.

**List of activated apps:**

Enabled:
 - accessibility: 1.6.0
 - activity: 2.13.4
 - admin_audit: 1.10.0
 - apporder: 0.11.0
 - audioplayer: 3.0.0
 - audioplayer_editor: 0.3.0
 - bruteforcesettings: 2.0.1
 - calendar: 2.1.2
 - circles: 0.20.6
 - cloud_federation_api: 1.3.0
 - cms_pico: 1.0.12
 - comments: 1.10.0
 - contacts: 3.4.2
 - contactsinteraction: 1.1.0
 - cospend: 1.2.4
 - dav: 1.16.2
 - deck: 1.2.2
 - drawio: 0.9.8
 - extract: 1.2.5
 - facerecognition: 0.7.2
 - federatedfilesharing: 1.10.2
 - federation: 1.10.1
 - files: 1.15.0
 - files_automatedtagging: 1.10.1
 - files_clipboard: 0.7.1
 - files_external: 1.11.1
 - files_pdfviewer: 2.0.1
 - files_photospheres: 1.20.0
 - files_rightclick: 0.17.0
 - files_sharing: 1.12.1
 - files_texteditor: 2.14.0
 - files_trashbin: 1.10.1
 - files_versions: 1.13.0
 - files_videoplayer: 1.9.0
 - firstrunwizard: 2.9.0
 - forms: 2.0.4
 - gpxpod: 4.2.4
 - groupfolders: 8.2.0
 - health: 0.1.3
 - imageconverter: 1.2.1
 - impersonate: 1.7.0
 - integration_google: 0.1.2
 - issuetemplate: 0.7.0
 - logreader: 2.5.0
 - lookup_server_connector: 1.8.0
 - maps: 0.1.8
 - metadata: 0.12.0
 - music: 0.17.3
 - news: 15.1.0
 - nextbackup: 20.9.0
 - nextcloud_announcements: 1.9.0
 - notes: 4.0.1
 - notifications: 2.8.0
 - oauth2: 1.8.0
 - password_policy: 1.10.1
 - photos: 1.2.1
 - polls: 1.6.2
 - previewgenerator: 3.0.1
 - privacy: 1.4.0
 - provisioning_api: 1.10.0
 - quicknotes: 0.6.3
 - richdocuments: 3.7.11
 - richdocumentscode_arm64: 6.4.202
 - serverinfo: 1.10.0
 - settings: 1.2.0
 - sharebymail: 1.10.0
 - social: 0.4.2
 - spreed: 10.0.4
 - support: 1.3.0
 - survey_client: 1.8.0
 - systemtags: 1.10.0
 - talk_simple_poll: 1.2.0
 - tasks: 0.13.6
 - text: 3.1.0
 - theming: 1.11.0
 - theming_customcss: 1.7.0
 - timemanager: 0.1.6
 - timetracker: 0.0.45
 - twofactor_backupcodes: 1.9.0
 - twofactor_nextcloud_notification: 3.0.0
 - twofactor_totp: 5.0.0
 - updatenotification: 1.10.0
 - user_status: 1.0.1
 - video_converter: 0.1.5
 - viewer: 1.4.0
 - weather_status: 1.0.0
 - whiteboard: 0.0.3
 - workflowengine: 2.2.0
Disabled:
 - breezedark
 - camerarawpreviews
 - checksum
 - cookbook
 - dashboard
 - data_request
 - duplicatefinder
 - encryption
 - epubreader
 - event_update_notification
 - files_3d
 - files_markdown
 - files_readmemd
 - files_retention
 - gpxedit
 - mail
 - ocsms
 - onlyoffice
 - phonetrack
 - recommendations
 - scanner
 - sharerenamer
 - user_ldap
 - user_usage_report
 - weather
 - workflow_pdf_converter
 - workflow_script


**Nextcloud configuration:**

{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "filelocking.enabled": true,
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***"
    ],
    "enable_previews": true,
    "preview_max_x": 1920,
    "preview_max_y": 1080,
    "enabledPreviewProviders": {
        "0": "OC\\Preview\\PNG",
        "1": "OC\\Preview\\JPEG",
        "2": "OC\\Preview\\GIF",
        "3": "OC\\Preview\\Image",
        "5": "OC\\Preview\\Epub",
        "6": "OC\\Preview\\OpenDocument",
        "7": "OC\\Preview\\Font",
        "11": "OC\\Preview\\Illustrator",
        "12": "OC\\Preview\\Postscript",
        "13": "OC\\Preview\\Photoshop",
        "14": "OC\\Preview\\TIFF",
        "15": "OC\\Preview\\MarkDown",
        "16": "OC\\Preview\\Movie",
        "17": "OC\\Preview\\Font"
    },
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "version": "20.0.4.0",
    "installed": true,
    "auth.bruteforce.protection.enabled": true,
    "appstoreenabled": true,
    "apps_paths": [
        {
            "path": "\/var\/www\/***REMOVED SENSITIVE VALUE***\/apps",
            "url": "\/apps",
            "writable": true
        }
    ],
    "forcessl": true,
    "theme": "",
    "maintenance": false,
    "mail_smtpmode": "smtp",
    "mail_smtpsecure": "ssl",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "forceSSLforSubdomains": true,
    "appstore.experimental.enabled": true,
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "loglevel": 0,
    "memcache.local": "\\OC\\Memcache\\APCu",
    "minimum.supported.desktop.version": "2.0.0",
    "quota_include_external_storage": false,
    "overwrite.cli.url": "https:\/\/cloud.tomaskaluza.net",
    "mail_smtpauthtype": "PLAIN",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_smtpport": "465",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "htaccess.RewriteBase": "\/",
    "dbtype": "mysql",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "updater.release.channel": "stable",
    "mysql.utf8mb4": true,
    "app_install_overwrite": [
        "calendar",
        "files_clipboard",
        "radio",
        "weather",
        "files_photospheres",
        "video_converter",
        "files_readmemd",
        "files_external_onedrive",
        "metadata",
        "forms",
        "social",
        "ocr",
        "breezedark"
    ],
    "has_rebuilt_cache": true
}
### Client configuration
**Browser:**
Google Chrome 87
**Operating system:**
Win 10, Ubuntu 20.04, Android 7.0, Android 10

**Comment by skjnldsv** [comment]
Thanks!

Could it be that you're talking about the top/left/right/bottom padding around the pdf? 
![image](https://user-images.githubusercontent.com/14975046/102897260-010f4e80-4468-11eb-8a5c-632abe8bb4cf.png)

**Comment by juliusbr** [comment]
I had the same problem with the white space at the bottom. I've tried disabling apps and disabling the Photo Sphere Viewer app solved this issue for me.

**Comment by elpraga** [comment]
I can confirm @juliusbr 's observation. Disabling **Photo Sphere Viewer** app fixes the issue.  I will file a corresponding bug report.

**Comment by elpraga** [comment]
Thanks everyone for looking into this issue, and especially @juliusbr for finding the solution!!

**Comment by R0Wi** [comment]
This is indeed a bug inside the `Photosphere Viewer` app. Will be fixed there https://github.com/nextcloud/files_photospheres/issues/68.

**Comment by elpraga** [comment]
Fixed in Photo Sphere Viewer version 1.20.1
